app-network/mihomo-party/autobuild/build

```diff
@@ -0,0 +1,36 @@
+abinfo "Fetching dependencies and prepare ..."
+export CI=true
+pnpm install
+sed -i "s/productName: Mihomo Party/productName: mihomo-party/" "$SRCDIR"/electron-builder.yml
```

Comment:
No `sed` please (we won't know whether the operation succeeded, or even worked expectedly), this should be a patch.